oshdb-api/pom.xml

```diff
@@ -97,19 +97,6 @@
         <artifactId>maven-jar-plugin</artifactId>
         <executions>
```

Comment:
We also miss the plugin version here. 
At the moment this is the only place where the ***maven-jar-plugin*** is used, so we could specify the version directly here or try to use the maven [pluginManagment](https://maven.apache.org/pom.html#Plugin_Management) mechanism in the ohsome-parent. 
Maybe it would also be worth checking if we could get rid of the version property definitions in the [ohsome-parent](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/parent/-/blob/master/pom.xml#L72) and user the maven [dependecyManagment](https://maven.apache.org/pom.html#Dependency_Management) instead.

Reply:
I added the `maven-jar-plugin` version in d29b4a8. I would like to separate the change to pluginManagement / dependencyManagement mechanisms into another PR.

Reply:
thank you! and yes we should do an other pr (if we like to) for the change to pluginManagement/dependencyManagement